lib/Models/CatalogGroup.js

```diff
@@ -367,10 +340,25 @@ CatalogGroup.prototype.toggleOpen = function() {
 };
 
 /**
+ * Finds the first item in this group with a value in {@link CatalogItem#allShareKeys} that matches the given key.
+ * The search is case-sensitive.
+ *
+ * @param {String} key The key to search for.
+ * @return {CatalogMember} The first item with the given key, or undefined if no item with that name exists.
+ */
+CatalogGroup.prototype.getChildForShareKey = function(key) {
+    return this.items.find(function(item) {
```

Comment:
`find` doesn't work in IE, unfortunately.  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find